.project-keeper.yml

```diff
@@ -0,0 +1,10 @@
+sources:
+  - type: maven
+    path: pom.xml
+    modules:
+      - integration_tests
+      - lombok
```

Comment:
does it use lombok? In the old version it did not declare the lombok module

Reply:
removed it. you're right, it doesn't use lombok afaik.